### PR TITLE
Remove a duplicated gce-1.7 job from prow config

### PR DESCRIPTION
**Pull request by krzyzacy**
Also add a test to avoid duplicated jobs of a same type.

### CODE REVIEW

File: prow/config/jobs_test.go [hunk 1]
```diff
@@ -524,3 +524,34 @@ func TestValidPodNames(t *testing.T) {
...
+func TestNoDuplicateJobs(t *testing.T) {
+	c, err := Load("../config.yaml")
+	if err != nil {
+		t.Fatalf("Could not load config: %v", err)
+	}
+
+	allJobs := make(map[string]bool)
+	for _, j := range c.AllPresubmits() {
+		if allJobs[j] == true {
+			t.Errorf("Found duplicate job %s.", j)
```
Comment (spiffxp) [inline prow/config/jobs_test.go]: call out presubmit vs. postsubmit vs. periodic in the error message?

Comment (krzyzacy) [inline prow/config/jobs_test.go]: done

**Comment by krzyzacy** [comment]
/assign @caesarxuchao 
it should make https://k8s-testgrid.appspot.com/release-1.7-blocking#gce-release-1-7 look better

**Comment by krzyzacy** [comment]
/assign @dchen1107 
tbr

**Comment by krzyzacy** [comment]
lgtm-ed from @caesarxuchao from offline :-)

**Comment by dchen1107** [comment]
/lgtm

Thanks!

**Pull request merged**